### DISCUSSION TITLE
SignalR Core and streaming

### DISCUSSION THREAD

**Discussion by YaroslavMudryk** [dotnet/aspnetcore#25680] [General] [answered]
Does SignalR Core support streaming? For an example video conferencing?

**Comment by ghost** [comment, marked as answer]
Why not [https://weblogs.asp.net/ricardoperes/video-streaming-with-asp-net-signalr-and-html5](https://weblogs.asp.net/ricardoperes/video-streaming-with-asp-net-signalr-and-html5)

**Reply by YaroslavMudryk** [reply]
Thanks)

**Reply by ghost** [reply]
@Yaroslav08 Are you work on it?

> Thanks)

**Reply by YaroslavMudryk** [reply]
no, I need to study this for a future project, but I looked like workers
